sale_timesheet_project_foresee/views/sale_order.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="sale_order_form_view">
+        <field name="name">sale.order.form (in sale_order_project_foresee)</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+
+            <button name="action_confirm" position="before">
+                <field name="action_project_foresee_allowed" invisible="1"/>
+                <button name="action_project_foresee" type="object"
+                        string="Project/Task Foresee"
```

Comment:
Foresee as a button name is not immediately obvious for me. "Generate Project/Tasks" instead?

Reply:
Yes it's more explicit indeed. I changed it.